document-render-blocking.md

```diff
@@ -36,16 +75,104 @@ The timing for when this attribute can block rendering for a Document is already
 </html>
 ```
 
-### Block Rendering on Partial Document Parsing
+#### Block Rendering on Partial Document Parsing
+
 ```html
 <html blocking="render">
+<script>
+  // The set of element IDs that should block rendering.
+  let blockingElementIds = new Set();
+
+  function maybeUnblockRendering() {
+    if (blockingElementIds.size == 0) {
+      document.documentElement.blocking="";
+    }
+  }
+
+  // The value returned by getState() is set by the old Document in
+  // the `navigate` event. It tracks whether the old Document add a
+  // `view-transition-name` to the header.
+  if (navigation.initialLoad.from().getState().morphHeader) {
+    blockingElementIds.add("header-id");
+  }
+  maybeUnblockRendering();
+</script>
 <body>
 …
+<div id="header-id">
+  ...
+</div>
 <script>
-   if (doPartialBlocking) {
-      document.documentElement.blocking="";
-   }
+  // When an element is parsed, remove it from the blocking set and
+  // unblock rendering if all blocking elements have been parsed.
+  blockingElementIds.delete("header-id");
+  maybeUnblockRendering();
+</script>
+</body>
+</html>
+```
+
+## Meta Tag with Blocking ElementIds
+
+Add a new meta tag with the name `blocking-elements` and `content` attribute set to the list of [element IDs](https://dom.spec.whatwg.org/#concept-id) which must be parsed before rendering. `*` is a special keyword which implies every ID should be blocking.
```

Comment:
```suggestion
Add a new meta tag with the name `blocking-elements` and `content` attribute set to the list of [element IDs](https://dom.spec.whatwg.org/#concept-id) which must be parsed before rendering. `*` is a special keyword which implies every element should be blocking.
```
The distinction is that if I don't specify an id on my elements, we should still block under `*` right?